### PR TITLE
OSPP 2021-结项报告-基于 Matrix AppService Wechaty 项目的用户、群与消息等基础功能的完善

### PR DESCRIPTION
**Pull request by lprintf**
Please make sure to check the following boxes creating your pull request, thank you very much!

- [x] Add Blog Header, including title, author, image, categories, tags, etc.  

    For example:  

    ```md
    ---
    title: Your Great Blog Title
    author: your_github_id
    categories: project
    tags:
      - python
      - padlocal
    image: /assets/2021/02-your-blog-slug/teaser.jpg
    ---

    One line bio

    Your blog post content Your blog post content
    Your blog post content Your blog post content
    Your blog post content Your blog post content
    ```

    > It will be easier to start drafting your post by using a existing post file as your template, for example, [this one](https://github.com/wechaty/wechaty.js.org/blob/master/jekyll/_posts/2021-01-14-wechaty-puppet-service.md)

- [x] Keep all filenames lowercase, and use `-` to connect words instead of space. e.g. `2017-10-06-wechat-pc-impactor.md` instead of `2017-10-06-WeChat PC Impactor.md`
- [x] Embed video/pdf from YouTube/Bilibili/local (if there's any) by using `{% include iframe.html src="URL_or_PATH" %}`. (Learn more about how to embed file/url in the post by reading this [blog post](https://wechaty.js.org/2020/08/24/add-video-to-wechaty-blog/)
- [x] Save photo to local before publishing, to a folder named by `${MONTH}-${YOUR_BLOG_SLUG}/` under `/assets/2021/` directory.
- [x] Select a beautiful and meaningful teaser image for your blog
- [x] Create your contributor profile (if you are a first time contributor)

    For excample:

    > jekyll/_contributors/lijiarui.md

    ```yaml
    ---
    name: Jiarui LI (李佳芮)
    site: https://lijiarui.github.io
    bio: 句子互动创始人 & CEO，微软人工智能最具价值专家 (AI MVP)
    avatar: /jekyll/assets/contributors/lijiarui/avatar.png
    email: rui@juzi.bot
    twitter: https://weibo.com/u/2175505900
    ---
    ```

  Here is the example file:

  - lijiarui.md: <https://github.com/wechaty/wechaty.js.org/blob/master/jekyll/_contributors/lijiarui.md>
  - It will shown like this on wechaty website: <https://wechaty.js.org/contributors/lijiarui/>

To learn more about the contribute guideline, visit: <https://wechaty.js.org/docs/contributor-program/>

### CODE REVIEW

**Review by lijiarui — CHANGES_REQUESTED** [review]
Good job！ Looking forward to your video link and your mentor's approval, then I will begin to review this PR.

**Comment by lijiarui** [comment]
Please DO the following steps first:

1. Get your mentor's approval first.
2. Link back to your ospp 2021 project issue.

@huan

File: jekyll/_posts/2021-09-28-ospp-final-term-matrix-wechaty.md [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+  - matrix
+  - summer-of-wechaty
+  - summer-2021
+  - ospp
+  - ospp-2021
+  - final-term
+  - ecosystem
+---
+
+“[开源软件供应链点亮计划-暑期2021](https://summer.iscas.ac.cn)”（以下简称 暑期2021）是由中科院软件所与 openEuler 社区共同举办的一项面向高校学生的暑期活动。旨在鼓励在校学生积极参与开源软件的开发维护，促进国内优秀开源软件社区的蓬勃发展。活动联合各大开源社区，针对重要开源软件的开发与维护提供项目，并向全球高校学生开放报名。 学生可自主选择感兴趣的项目进行申请，并在中选后获得该软件资深维护者（社区导师）亲自指导的机会。 根据项目的难易程度和完成情况，参与者还可获取“开源软件供应链点亮计划-暑期2021”活动奖金和奖杯。
```
Comment (huan) [inline jekyll/_posts/2021-09-28-ospp-final-term-matrix-wechaty.md]: 1. Please add the project introduction at the beginning of this post
2. Please move the OSPP 2021 introduction to the end of this post.

**Review by huan — APPROVED** [review]
LGTM

Thank you very much for your great video presentation, it's very clear and I believe it will help readers understand what you have done with the project clearly!

**Comment by lprintf** [comment]
https://github.com/wechaty/summer/issues/59

**Review by lijiarui — APPROVED** [review]
Good job!

**Comment by huan** [comment]
link to

- https://github.com/wechaty/summer/issues/59

**Pull request merged**